spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/PollerProperties.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.integration.scheduling.PollerMetadata;
+import org.springframework.scheduling.support.PeriodicTrigger;
+
+/**
+ * @author Dave Syer
+ *
+ */
+@ConfigurationProperties("spring.integration.poller")
+public class PollerProperties {
+
+	/**
+	 * Fixed delay for default poller
+	 */
+	private Long fixedDelay = 1000L;
```

Comment:
I was planning to support other trigger types (with null meaning "not set"). Leaving that for later now, and unwrapping the longs.